MOBILE_API.xml

```diff
@@ -2694,6 +2779,8 @@
         <element name="GetAppServiceDataID" value="53" hexvalue="35" since="5.1" />
         <element name="GetFileID" value="54" hexvalue="36" since="5.1" />
         <element name="PerformAppServiceInteractionID" value="55" hexvalue="37" since="5.1" />
+        <element name="CreateWindowID" value="60" hexvalue="3C" since="6.0" />
+        <element name="DeleteWindowID" value="61" hexvalue="3D" since="6.0" />
```

Comment:
These lines are implied by the additions here: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0216-widget-support.md#mobile-api.

The other hex values just must not have been merged yet. Here is what i currently have in core for 6.0

```
        <element name="UnpublishAppServiceID" value="56" hexvalue="38" since="6.0" />
        <element name="CancelInteractionID" value="57" hexvalue="39" since="6.0" />
        <element name="CloseApplicationID" value="58" hexvalue="3A" since="6.0" />
        <element name="ShowAppMenuID" value="59" hexvalue="3B" since="6.0" />
        <element name="CreateWindowID" value="60" hexvalue="3C" since="6.0" />
        <element name="DeleteWindowID" value="61" hexvalue="3D" since="6.0" />
        <element name="GetInteriorVehicleDataConsentID" value="62" hexvalue="3E" since="6.0" />
        <element name="ReleaseInteriorVehicleDataModuleID" value="63" hexvalue="3F" since="6.0" /> 
```